bin/deploy.sh

```diff
@@ -36,14 +37,18 @@ ZIP_FILE=$VERSION.zip
 zip -r $ZIP_FILE Dockerrun.aws.json
 aws s3 cp $ZIP_FILE s3://$EB_BUCKET/$APP_NAME/$ZIP_FILE
 
-# In production, create a new app version
-if [ "$ENV" == "live" ]
-then
-  # Create a new application version with the zipped up Dockerrun file
-  aws elasticbeanstalk create-application-version --application-name $APP_NAME \
-      --version-label $VERSION --source-bundle S3Bucket=$EB_BUCKET,S3Key=$APP_NAME/$ZIP_FILE --region $AWS_REGION
-fi
+# Create a new application version with the zipped up Dockerrun file
+aws elasticbeanstalk create-application-version \
+  --application-name $APP_NAME \
+  --version-label $VERSION \
+  --source-bundle S3Bucket=$EB_BUCKET,S3Key=$APP_NAME/$ZIP_FILE \
+  --region $AWS_REGION
 
 # Update the environment to use the new application version
-aws elasticbeanstalk update-environment --environment-name $APP_NAME-$ENV \
-    --version-label $VERSION --region $AWS_REGION
+aws elasticbeanstalk update-environment \
+  --environment-name $APP_NAME-$ENV \
+  --version-label $VERSION \
+  --region $AWS_REGION
+
+echo
+echo Done! AWS EB deployment rollout in progress.
```

Comment:
Just formatting for readability and a friendly success message since the previous version just dumped a load of json to the terminal which is not obviously a success.